StudCity.Frontend/src/components/SearchUsersForCreatedChats/index.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Tabs from "../Tabs";
+import { LABELS_USERS_GROUPS } from "../../utils/constants";
+import Scroller from "../Scroller";
+import { selectDataUsersFound, getUsersSearch } from "../../app/features/userSlice";
+import TextField from "../../UI/fields/TextField";
+import Spinner from "../Spinner";
+import Avatar from "../../UI/Avatar";
+
+export default function UserGroupSearch({ setSelectedUsers, selectedUsers }) {
+	const [tabIndex, setTabIndex] = useState(0);
+	const dispatch = useDispatch();
+	const [users, setUsers] = useState([]);
+	const [searchData, setSearchData] = useState('');
+	const [totalItemsCount, setTotalItemsCount] = useState(0);
+	const { data, loading } = useSelector(selectDataUsersFound);
+	const [indexPage, setIndexPage] = useState(1);
```

Comment:
We have `totalItemsCount`, `indexPage`, `searchData`, what do you think about adding one state object for this search user information? Because we use this in one context. 
@Serhiy-dotcom @Vova-Romaniuk ?

Reply:
I don't understand what context we are talking about